pkg/scheduler/actions/shuffle/shuffle.go

```diff
@@ -61,7 +61,7 @@ func (shuffle *Action) Execute(ssn *framework.Session) {
 	// Evict target workloads
 	victims := ssn.VictimTasks(tasks)
 	for victim := range victims {
-		klog.V(5).Infof("Victim %s: [ns: %s, job: %s]\n", victim.Name, victim.Namespace, victim.Job)
+		klog.V(3).Infof("Victim %s: [ns: %s, job: %s]\n", victim.Name, victim.Namespace, victim.Job)
```

Comment:
Make the log to be a complete sentence and easy to read for users.  e.g. "Job %s from namespace %s will be evited\n."

Reply:
OK.